aequilibrae/distribution/gravity_calibration.py

```diff
@@ -218,15 +218,15 @@ def __check_inputs(self):
         mats = [(self.matrix, "Observed matrix"), (self.impedance, "Impedance matrix")]
         for matrix, title in mats:
             if matrix.matrix_view is None:
-                raise ValueError(title + " needs to be set for computation")
+                raise ValueError("{title} needs to be set for computation")
             else:
                 if len(matrix.matrix_view.shape[:]) > 2:
```

Comment:
Maybe we could use something like `matrix.matrix_view.ndim > 2` ?

Reply:
agreed